policyengine_us/parameters/gov/states/ct/tax/income/main/add_back/increment.yaml

```
@@ -0,0 +1,22 @@
+description: Connecticut phases its 3% income tax rate out in these increments, based on filing status.
```

to avoid including parameter values in labels/descriptions (elsewhere too)
```suggestion
description: Connecticut phases its bottom income tax rate out in these increments, based on filing status.
```